trace-sdk/build.gradle

```diff
@@ -78,7 +77,7 @@ dependencies {
 }
 
 junitJacoco {
-    jacocoVersion = '0.8.6'
+    jacocoVersion = '0.8.3'
```

Comment:
Why was the version of it degraded?

Reply:
I used what's in the readme for https://github.com/vanniktech/gradle-android-junit-jacoco-plugin - i'll update it and see if it changes the reports.